composer.json

```diff
@@ -47,8 +48,10 @@
     "fakerphp/faker": "^1.21",
     "mockery/mockery": "^1.0",
     "nunomaduro/larastan": "^2.4",
+    "phpstan/extension-installer": "^1.3",
     "phpstan/phpstan": "^1.10",
     "phpstan/phpstan-deprecation-rules": "^1.1",
+    "phpstan/phpstan-mockery": "^1.1",
```

Comment:
These are necessarily related to the rest of this PR but it's probably alright to include them anyway